### PR TITLE
Create 0000-01-02-Zlchm.md

### PR DESCRIPTION
**Pull request by Lyrate**
Add Zlchm's file

### CODE REVIEW

File: _posts/0000-01-02-Zlchm.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Long time no see...hey ukulele
```
Comment (Lyrate) [inline _posts/0000-01-02-Zlchm.md]:
```suggestion wellness
Long time no see...hey ukulele
```

```

Comment (Zlchm) [inline _posts/0000-01-02-Zlchm.md:5]: hello~

Comment (Zlchm) [inline _posts/0000-01-02-Zlchm.md:5]: weeerrfdg,,fg

Comment (Lyrate) [inline _posts/0000-01-02-Zlchm.md]: ewwh

**Pull request merged**